### PR TITLE
Make LoadCollection compatible

### PR DESCRIPTION
**Pull request by XuanYang-cn**
See also: #17656

Signed-off-by: yangxuan <xuan.yang@zilliz.com>

/kind improvement

### CODE REVIEW

**Comment by XuanYang-cn** [comment]
rerun ut

**Comment by XuanYang-cn** [comment]
rerun ut

**Comment by XuanYang-cn** [comment]
rerun ut

File: internal/proxy/task.go [hunk 1]
```diff
@@ -2576,6 +2576,11 @@ func (lct *loadCollectionTask) PreExecute(ctx context.Context) error {
 		return err
 	}
 
+	// To compat with LoadCollcetion before Milvus@2.1
```
Comment (congqixia) [inline internal/proxy/task.go:2579]: NIT: typo

**Review by congqixia — APPROVED** [review]
/lgtm

**Pull request merged**